browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -462,126 +451,6 @@ IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "oauth=true");
-}
-
-IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
-                       ThirdPartyGoogleOauthCookieAllowAllBlock3pSiteOverride) {
-  DefaultAllowAllCookies();
-  BlockThirdPartyCookies(https_top_level_page_url_);
-  NavigateToPageWithFrame(https_cookie_iframe_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "");
-
-  NavigateFrameTo(google_oauth_cookie_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "oauth=true");
-}
-
-IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
-                       ThirdPartyGoogleOauthCookieAllowAllBlockSiteOverride) {
-  DefaultAllowAllCookies();
-  BlockCookies(https_top_level_page_url_);
-  NavigateToPageWithFrame(https_cookie_iframe_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "");
-
-  NavigateFrameTo(google_oauth_cookie_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "oauth=true");
-}
-
-IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
-                       ThirdPartyGoogleOauthCookieBlockAllAllowSiteOverride) {
-  DefaultBlockAllCookies();
-  AllowCookies(https_top_level_page_url_);
-  NavigateToPageWithFrame(https_cookie_iframe_url_);
-  ExpectCookiesOnHost(https_top_level_page_url_, "name=Good");
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "");
-
-  NavigateFrameTo(google_oauth_cookie_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "oauth=true");
-}
-
-IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
-                       ThirdPartyGoogleOauthCookieBlockAllBlock3pSiteOverride) {
-  DefaultBlockAllCookies();
-  BlockThirdPartyCookies(https_top_level_page_url_);
-  NavigateToPageWithFrame(https_cookie_iframe_url_);
-  ExpectCookiesOnHost(https_top_level_page_url_, "name=Good");
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "");
-
-  NavigateFrameTo(google_oauth_cookie_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "oauth=true");
-}
-
-IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
-                       ThirdPartyGoogleOauthCookieBlockAllBlockSiteOverride) {
-  DefaultBlockAllCookies();
-  BlockCookies(https_top_level_page_url_);
-  NavigateToPageWithFrame(https_cookie_iframe_url_);
-  ExpectCookiesOnHost(https_top_level_page_url_, "");
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "");
-
-  NavigateFrameTo(google_oauth_cookie_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "oauth=true");
-}
-
-IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
-                       ThirdPartyGoogleOauthCookieBlocked) {
-  BlockGoogleOAuthCookies();
-  NavigateToPageWithFrame(https_cookie_iframe_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "");
-
-  NavigateFrameTo(google_oauth_cookie_url_);
-  ExpectCookiesOnHost(GURL("https://accounts.google.com"), "");
-}
-
```

Comment:
why these tests are removed? they should stay and we should run them with the new feature disabled.

Reply:
I just moved them to the new browsertest class I added, and I do exactly that: run them with the feature flag off. I figured would be best to keep all tests related to this feature in the same place.

Reply:
Great. GitHub didn't show the big diff for that new browsertest file, that's why I haven't found them using a simple Ctrl+F. Sorry! :)